### ISSUE TITLE
Add pagination

### ISSUE THREAD

**Issue by kbuika** [kibuikaCodes/tusome-v2#8]

**Comment by HenryKenya** [comment]
Is this issue still open?

**Comment by kibuikaCodes** [comment]
Yes, it is. Go ahead, I'll appreciate

**Comment by HenryKenya** [comment]
I'll try make a PR on it then

**Comment by HenryKenya** [comment]
Is it paginated on the backend as well? And is there some API docs I can have a look at?

**Comment by kibuikaCodes** [comment]
Yes, it is. I was relying on a postman collection documentation that doesn't seem to have that pagination request, but I know it is there, let me get it then I'll send it in a few hours.

**Comment by HenryKenya** [comment]
Cool.

**Comment by kibuikaCodes** [comment]
Hey @HenryKenya, the backend guy @willynganga is having issues with swagger but basically, this is how the pagination is happening.

https://tusome-app.herokuapp.com/api/v1/papers/all?page=1
It returns an array with the total pages, current page number and the papers in that page.

I am sure you'll notice that the API route is different from the current one, replace it with this one. 
Thanks

**Comment by HenryKenya** [comment]
Alright. Will have a look at it

**Comment by kibuikaCodes** [comment]
Fantastic
